kibbeh/src/ui/ProfileTabs.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+export interface ProfileTabsProps {
+   activeTab: string
+};
+
+export const ProfileTabs: React.FC<ProfileTabsProps> = ({ activeTab }) => {
+   return (
+      <div className="w-full flex items-center justify-around">
+         <button 
+            className={`py-1 text-primary-100 text-base font-bold hover:text-accent-hover 
```

Comment:
Change hover effect to add border instead of changing the color.

```suggestion
            className={`py-1 text-primary-100 text-base font-bold border-b-2 border-primary-900 hover:border-accent transition
```

Reply:
Alright. Noted

Reply:
`border-primary-900` is a white border tho. Did you mean `border-accent`?

Reply:
`border-primary-900` is the darkest gray we have. What I did here is match the border color with the background one (so it looks invisible) and then show the `accent` on hover.

Reply:
Ok my bad. Should be fine now